frontend/src/components/transaction-status-code/TransactionStatusCode.module.scss

```diff
@@ -28,7 +28,7 @@
   }
 
   &.sealed {
-    color: $color-status-sealed;
+    color: #A2CD8C;
```

Comment:
Is this some predefined color? If not, can we add it to a shared colors file (`styles/colors.scss`) and set some logical generic name (e.g. light blue, dark green, lighter red,...)?

Reply:
I think that it's better to keep all of our color definitions centralised in a single file, because that encourages reusability and it's easier to change in case some colors change.